Platform/ElkhartlakeBoardPkg/CfgData/CfgData_Silicon.yaml

```diff
@@ -1857,21 +1857,43 @@
       option       : $EN_DIS
       help         : >
                      When FALSE, it disables Timed GPIO 0.
-      length       : 1b
+      length       : 0x01
       value        : 0x0
   - EnableTimedGpio1 :
       name         : Timed GPIO 1
       type         : Combo
       option       : $EN_DIS
       help         : >
                      When FALSE, it disables Timed GPIO 1.
-      length       : 1b
+      length       : 0x01
       value        : 0x0
+  - PchTsnGbeSgmiiEnable :
+      name         : PCH TSN SGMII Support
+      type         : Combo
+      option       : $EN_DIS
+      help         : >
+                     Enable/disable PCH TSN SGMII support 0=Disable 1=Enable
+      length       : 0x01
+      value        : 0x1
+  - PseTsnGbeSgmiiEnable :
+      name         : PSE TSN SGMII Support
+      type         : EditNum, HEX, (0x00,0xFFFF)
+      help         : >
+                     Enable/disable PSE TSN SGMII support 0=Disable 1=Enable
+      length       : 0x2

```

Comment:
Why do you want to use 2 byte  instead of 1 bit for ENABLE/DISABLE?

Reply:
Why EditNum type ?

Reply:
if you have two interfaces, then it should be broken into 2 item and each is a single bit.

Reply:
i saw there are some similar multiple interface cfgdata with enable/disable option are using EditNum with bytes as well. 
Example:
  - SataPortsEnable :
      name         : Enable SATA ports
      type         : EditNum, HEX, (0x00,0xFFFFFFFFFFFFFFFF)
      help         : >
                     Enable/disable SATA ports. One byte for each port, byte0 for port0, byte1 for port1, and so on.
      length       : 0x08
      value        : { 0x01, 0x01, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00}
Should we change them as well?

Reply:
Yes, but I think it is more user friendly if we break it down into individual config item.